clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -241,11 +251,32 @@ def dataset_to_bids(complete_data_df: DataFrame, gif: bool) -> Dict[str, DataFra
         "genfi_ref.csv",
     )
     df_ref = pd.read_csv(path_to_ref_csv, sep=";")
-
     if not gif:
         df_ref = df_ref.head(8)
+    # add additional data through csv
+    if path_to_clinical_tsv:
+        additional_data_df = pd.read_csv(path_to_clinical_tsv, sep="\t")
+
+        path_to_mapping_tsv = os.path.join(
```

Comment:
I think we should use Path objects consistently across the code base rather than strings. Especially when it makes the code much easier to read. I would modify both `path_to_mapping_tsv` and `path_to_ref_csv`.

To be verified, but this would become:

```python
from clinica.utils.filemanip import get_parent
genfi_data_folder = get_parent(__file__, 3) / "data"
path_to_ref_csv = genfi_data_folder / "genfi_ref.csv"
...
path_to_mapping_tsv = genfi_data_folder / "genfi_data_mapping.tsv"
```